test/integration/targets/copy/tasks/tests.yml

```
@@ -92,6 +93,18 @@
       - "stat_results.stat.issock == false"
       - "stat_results.stat.checksum == ('foo.txt\n'|hash('sha1'))"
 
+- name: Repeat the basic copy
+  copy:
+    src: foo.txt
```

```suggestion
    content: other_data
```
Updated content with different data to make sure `force: false` is working (should not replace content if the dest exists).